makefile/src/com/jetbrains/lang/makefile/toolWindow/MakefileFileNode.kt

```diff
@@ -32,4 +34,9 @@ class MakefileFileNode(val psiFile: PsiFile, private val targets: List<MakefileT
   override fun getIndex(node: TreeNode) = targets.indexOf(node)
 
   override fun getAllowsChildren() = true
+
+  override fun renderName(renderer: ColoredTreeCellRenderer) {
+    super.renderName(renderer)
+    renderer.append("  ${psiFile.virtualFile.path}", SimpleTextAttributes.GRAYED_ITALIC_ATTRIBUTES, false)
```

Comment:
Is there any way to make this fragment right-aligned? I tried different values for `padding` of this and filename fragments together with `alignment = RIGHT` - no effect.